frontend/components/customizer/customizer.js

```diff
@@ -18,16 +18,12 @@ export default class Customizer extends React.Component {
     exampleNamedParams: objectOfKeyValuesPropType,
     exampleQueryParams: objectOfKeyValuesPropType,
     initialStyle: PropTypes.string,
+    showActualParams: PropTypes.bool,
```

Comment:
Could you call this something like `isPrefilled`?

Reply:
Thanks. Naming is a tough task :-)